### PR TITLE
fix(modal): adds overflow: hidden from code and fixes scrollbar measurement

### LINKED ISSUE #4128: Opened modal does not remove scrollbar of body (Bootstrap5)
When a modal is openend, the body scrollbar is still visible and the background is scrollable.

It looks like the `overflow:hidden` class for `body.modal-open` was removed and the stylings are applied in the js directly on the body

For an example, just run the demo on the bootstrap 5 branch an open any modal

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 12

ng-bootstrap: 11.0.0-beta.2

Bootstrap: 5.0.2

### PR DESCRIPTION
**Pull request by divdavem**
fix #4128

### CODE REVIEW

File: src/util/scrollbar.ts [hunk 1]
```diff
@@ -1,80 +1,52 @@
...
-    measurer.className = 'modal-scrollbar-measure';
-
-    const body = this._document.body;
-    body.appendChild(measurer);
-    const width = measurer.getBoundingClientRect().width - measurer.clientWidth;
-    body.removeChild(measurer);
-
-    return width;
+    const documentWidth = this._document.documentElement.clientWidth;
+    return Math.abs(window.innerWidth - documentWidth);
```
Comment (fbasso) [inline src/util/scrollbar.ts:50]: documentWidth is not necessary, it can be done in the same line

File: src/util/scrollbar.ts [hunk 2]
```diff
@@ -1,80 +1,52 @@
...
+      const actualPadding = parseFloat(window.getComputedStyle(body)['padding-right']);
+      body.style['padding-right'] = `${actualPadding + scrollbarWidth}px`;
+    }
+    body.style['overflow'] = 'hidden';
+    return () => {
+      if (scrollbarWidth > 0) {
+        body.style['padding-right'] = userSetPaddingStyle;
+      }
+      body.style.overflow = overflowStyle;
+    };
```
Comment (fbasso) [inline src/util/scrollbar.ts:40]: As before, avoiding repeating body.style is better.

**Review by fbasso — CHANGES_REQUESTED** [review]
Thanks for your PR @divdavem  ! 

Some little changes to consider...

**Comment by divdavem** [comment]
@fbasso Thank you for your review! It is good indeed to simplify code. I have done the suggested changes.

**Comment by fbasso** [comment]
Thanks @divdavem

**Comment by divdavem** [comment]
@fbasso Thank you for merging my PR

**Pull request merged**